src/offices/_single.html

```diff
@@ -15,7 +15,7 @@
 {% block content_main %}
 
     {% if office.title %}
-    <h1>{{ office.title | safe }}</h1>
+    <h1 class="qa-main-title">{{ office.title | safe }}</h1>
```

Comment:
If we're following proper headings now, there should only be one h1 on each page and qa class hooks shouldn't be needed here.